Management_System/DAL/Repositories/Interfaces/IRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DAL.Repositories.Interfaces;
+
+public interface IRepository<T> where T : class
```

Comment:
Please let do code style in one way.
Do we add blank line after namespace or not?

Fix all updated files please